README.md

```diff
@@ -96,7 +96,7 @@ await publisher.testExchange({someIdentifier: '...', routes: [], ...});
 ```
 
 ## Testing
-You'll need to fill a file called `taskcluster-base-test.conf.json` with valid keys. There is a `taskcluster-base-test-example.conf.json` you can copy over to see which keys are needed. Then it is just a matter of `npm install` and `npm test`.
+You'll need to fill a file called `taskcluster-base-test.conf.json` with valid keys. There is a `taskcluster-base-test-example.conf.json` you can copy over to see which keys are needed. Then it is just a matter of `yarn install` and `yarn test`.
```

Comment:
Looks like the `user-config.yml` is what you need... But that is unrelated to this change... But yes, we should update readme...